atomic_reactor/schemas/config.json

```diff
@@ -706,6 +706,25 @@
         },
         "required": ["allowed_registries"],
         "additionalProperties": false
+    },
+    "build_env": {
```

Comment:
Maybe this name is too generic. How about "build_env_vars" ?

Reply:
Ok, changed